packages/config-vite-base/src/setup.ts

```diff
@@ -0,0 +1,5 @@
+// FAIL LOUDLY on unhandled promise rejections / errors
```

Comment:
Adding this because vitest fails if an error is uncaught but no trace is given this will add that.